webots_ros2_tiago/launch/robot_launch.py

```diff
@@ -106,14 +107,17 @@ def generate_launch_description():
         condition=launch.conditions.IfCondition(use_rviz)
     )
 
-    nav2 = IncludeLaunchDescription(
-        PythonLaunchDescriptionSource(os.path.join(get_package_share_directory('nav2_bringup'), 'launch', 'bringup_launch.py')),
-        launch_arguments=[
-            ('map', nav2_map),
-            ('use_sim_time', use_sim_time),
-        ],
-        condition=launch.conditions.IfCondition(use_nav)
-    )
+    if 'nav2_bringup' in get_packages_with_prefixes():
+        optional_nodes.append(IncludeLaunchDescription(
+            PythonLaunchDescriptionSource(os.path.join(
+                get_package_share_directory('nav2_bringup'), 'launch', 'bringup_launch.py')),
+            launch_arguments=[
+                ('map', nav2_map),
+                ('use_sim_time', use_sim_time),
+            ],
+            condition=launch.conditions.IfCondition(use_nav)
+        )
```

Comment:
Indentation looks wrong here:
```suggestion
            )
```